test/integration/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "integration",
+  "version": "1.0.0",
+  "description": "Integration tests for the Rev.ai node sdk",
+  "main": "index.js",
+  "jest": {
+    "testEnvironment": "node"
+  },
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/revdotcom/revai-node-sdk.git"
+  },
+  "author": "Rev Ai",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/revdotcom/revai-node-sdk/issues"
+  },
+  "homepage": "https://github.com/revdotcom/revai-node-sdk#readme",
```

You can put the link to the integration sub project
```suggestion
  "homepage": "https://github.com/revdotcom/revai-node-sdk/test/integration",
```